tutorials/preprocessing/70_fnirs_processing.py

```diff
@@ -26,6 +26,24 @@
 raw_intensity.load_data()
 
 
+###############################################################################
+# Providing more meaningful annotation information
+# ------------------------------------------------
+#
+# First, we attribute more meaningful names to the trigger codes which are
+# stored as annotations. Second, we include information about the duration of
+# each stimulus, which was 5 seconds for all conditions in this experiment.
+# Third, we remove the trigger code 15, which signaled the start and end
+# of the experiment and is not relevant to analysis.
+
+raw_intensity.annotations.set_durations(5)
+raw_intensity.annotations.rename({'1.0': 'Control',
+                                  '2.0': 'Tapping/Left',
+                                  '3.0': 'Tapping/Right'})
+raw_intensity.annotations.delete(
+    raw_intensity.annotations.description == '15.0')
```

Comment:
```suggestion
raw_intensity.annotations.delete(
    raw_intensity.annotations.description == '15.0'
)
```
If you use this style, I think the closing parenthesis should go into a new line (at least that's what black does).

Reply:
That's what `black` does and the way I prefer it too; but in MNE-Python, the dominant style is the one @rob-luke proposed. 

@cbrnr Would you like to make an "official" proposal to adopt the `black` style? You'd have my support.

Reply:
I never saw that style; I only know these two variants:

```
some_function(
    x,
    y,
    z
)

some_function(x,
              y,
              z)
```

Do we really use a mixture of the two?

Re black style, not sure if I'd want to go full in. Sometimes black formatting looks weird.

Reply:
-1 on this suggestion. Most places we deviate from `black` and keep the closing paren on the end, so I'd rather keep doing it that way at least for now. Or at least not force @rob-luke to deviate from our current practices.

@hoechenberger feel free to open a separate issue to discuss `black` adoption and/or if we should change our style practices/standards, let's not do it here.

Reply:
> I never saw that style

really it's all over the place in MNE's sources … I always though it looks very ugly :)

Reply:
> @hoechenberger feel free to open a separate issue to discuss `black` adoption and/or if we should change our style practices/standards, let's not do it here.

I'll leave that to @cbrnr :)

Reply:
> really it's all over the place in MNE's sources … I always though it looks very ugly :)

Can you show an example? I have never seen using a newline after `(`, indenting one level, and the putting `)` in the same line. We normally indent up to `(` and not just 4 spaces.

Reply:
It seems like sometimes we do use it (e.g. https://github.com/mne-tools/mne-python/blob/f1e6097c36e7232514d67d4b04af568e36ac5758/mne/beamformer/_compute_beamformer.py#L226-L228), but it is much less common. I'm fine with having all these variants, I just thought it was a mistake.